src/common/models/data-cube/data-cube.ts

```diff
@@ -704,8 +704,6 @@ export class DataCube implements Instance<DataCubeValue, DataCubeJS> {
     // No need for the overrides
     value.attributeOverrides = null;
 
-    value.options = null;
```

Comment:
I'm only worried what's inside options.druidContext - it's type is defined as opaque `Record` and we didn't use it for anything. Is there some sensitive data? Or could be? Maybe explicitly pass
``` 
  customAggregations?: CustomDruidAggregations;
 customTransforms?: CustomDruidTransforms;
```
?

@piter75 do you have any input?

Reply:
Druid context is also sent in query, right? In this case it would be needed as well. 
I don't think there's sensitive data in [context](http://druid.io/docs/latest/querying/query-context.html).
Since no authentication mechanism is implemented in Turnilo I guess you only expose the app inside a VPN (or using Basic auth). So only trusted people would access.

Reply:
http://druid.io/docs/latest/querying/query-context.html

Reply:
You're absolutely right!